bin/run

```diff
@@ -50,6 +50,22 @@ init_connections() {
     airflow connections --add \
         --conn_id amplitude_s3_conn \
         --conn_type s3
+
+    airflow connections --add \
```

Comment:
I'd prefer not to add these blank connections. When testing locally and not overriding these, the error messages were not obvious. E.g. I saw "TypeError: execv() arg 2 must contain only strings" when this conn wasn't overwritten

Reply:
It's a bit alarming to run dev and have a bunch of errors, plus IMHO it makes it easier to change them if they are already present. However that error is indeed not clear, any idea what it came from? I haven't seen anything like that.

Reply:
It comes when trying to run GKEPodOperator using the blank connection defined here

Reply:
Yeah, that's definitely not great. It doesn't look like we can add non-empty `google_cloud_platform` connections via the CLI. I'll open an issue in Airflow.

Reply:
or add dummy records?

Reply:
Yeah that's what I was trying to do, but there's no CLI spec for `json-key` as one of the connection variables.